apps/console/src/app/pages/projects/ProjectPage.tsx

```diff
@@ -6,10 +6,12 @@ import { useState } from "react";
 import { GettingStartedWizard } from "../../components/getting-started-wizard/GettingStartedWizard";
 import { GettingStartedWizardProvider } from "../../lib/providers/GettingStartedWizardProvider";
 import { Loader } from "../../components/common/Loader";
+import { usePageTitle } from "../../lib/hooks/usePageTitle";
 
 export const ProjectPage = () => {
   const { project, isLoading } = useCurrentProject();
   const [clicked, setClicked] = useState(false);
+  usePageTitle(project?.name || "Project");
```

Comment:
```suggestion
  usePageTitle(project?.name);
```
With the fallback we don't need the `||`